lib/active_record/connection_adapters/mysql2rgeo/schema_statements.rb

```diff
@@ -39,7 +39,7 @@ def create_table_definition(*args, **options)
         end
 
         # override
-        def new_column_from_field(table_name, field)
+        def new_column_from_field(table_name, field, _definitions)
```

Comment:
Required by this: https://github.com/rails/rails/pull/48241/files#diff-d1bec6fcf0cf6444eb5c2227f551c124419c572f1615aacf4a9894da4412f5a4R178